### ISSUE TITLE
Improve codebase 

### ISSUE THREAD

**Issue by DeveloperAspire** [DeveloperAspire/developers-fact-card#28]
The codebase is written with vanilla HTML, and it is getting larger as more people contribute to it. This has been harmful in the long run, it has led to some issues like merge conflicts, deleted codes and many more.

The codebase needs to be enhanced, it will still be basic and readable for beginners but also dynamic so the copy and paste will be reduced.

A framework can be used to achieve this.

After much scouting, I think Apline is the best tool for the job.

The tailwind codes also needs to be updated. 

The different quotes and their details transferred to a Config file( where users will make their contributions to.)

**Comment by kaf-lamed-beyt** [comment]
For a config file... would a `data.js` file with an array of objects suffice? 

One would just use `map` to render the items with a single markup. 

The cards remain the same. The tailwind classes too are retained (a single instance), if it is preferable.

**Comment by DeveloperAspire** [comment]
@kaf-lamed-beyt Yes yes, I was actually thinking of it too. Yup, this will suffice.

**Comment by DeveloperAspire** [comment]
@kaf-lamed-beyt the loop will be done via a dynamic framework e.g Apline. I would sit and learn how it works and probably handle that side of things.

**Comment by kaf-lamed-beyt** [comment]
> @kaf-lamed-beyt the loop will be done via a dynamic framework e.g Apline. I would sit and learn how it works and probably handle that side of things.

I'm guessing the use of the framework isn't only to fix the loop-ing of the markup, right?

**Comment by DeveloperAspire** [comment]
> > @kaf-lamed-beyt the loop will be done via a dynamic framework e.g Apline. I would sit and learn how it works and probably handle that side of things.
> 
> I'm guessing the use of the framework isn't only to fix the loop-ing of the markup, right?

Not only that, but to add a little bit of dynamic and state management where & when needed without necessarily using bulky frameworks or what do you think?

**Comment by kaf-lamed-beyt** [comment]
Hmmm🤔

What type of UI state do you intend on managing though. Can you give me an example?

Just to chip this in; unless the scope of would go beyond the facts card alone. I think vanillaJS might suffice.
